package.json

```
@@ -66,7 +66,8 @@
     "test:ftr:runner": "node scripts/functional_test_runner",
     "test:ftr:server": "node scripts/functional_tests_server",
     "test:jest": "node scripts/jest",
-    "test:jest_integration": "node scripts/jest_integration"
+    "test:jest_integration": "node scripts/jest_integration",
+    "typecheck": "node scripts/type_check"
```

To be consistent with our other script here, we should rename this like so:

```suggestion
    "test:type_check": "node scripts/type_check"
```